README_MACOSX_NOTES.md

```diff
@@ -132,7 +132,7 @@ FLUSH PRIVILEGES;
 exit;
 # Setting it to null above seems necessary. Then:
 mysql -u root
-ALTER USER 'root'@'localhost' IDENTIFIED WITH mysql_native_password BY 'root';
+ALTER USER 'root'@'localhost' IDENTIFIED BY 'root';
```

Comment:
I suggest removing the entire section starting with 
`@JoeCohen had problems setting the MySQL root password as follows:`
through
`7. Restart MySQL
brew services start mysql`